config_defaults_inc.php

```diff
@@ -2038,23 +2038,42 @@
 
 
 /**
- * Specifies the LDAP or Active Directory server to connect to.
+ * Specifies the LDAP or Active Directory server(s) to connect to.
+ * Multiple servers can be specified as a space-separated list.
  *
  * This must be a full LDAP URI (ldap[s]://hostname:port)
- * - Protocol can be either ldap or ldaps (for SSL encryption). If omitted,
- *   then an unencrypted connection will be established on port 389.
+ * - Protocol must be either: ldap (unencrypted or opportunistic StartTLS) or
+ *   ldaps (for TLS encryption).
```

Comment:
List the values as an indented, unordered list, e.g.
```
- Protocol must be one of: 
  - ldap (unencrypted or opportunistic StartTLS)
  - ldaps (for TLS encryption).
```

Reply:
Fixed.